docs/src/jll.md

```diff
@@ -209,3 +209,15 @@ defined for it are:
 
 * `data_txt_path`: this unexported variable is actually equal to `data_txt`, but
   is kept for consistency with all other product types.
+
+
+## Overriding `dev`'ed JLL packages
+
+In the event that a user wishes to override the binaries within a JLL package with
+their own binaries, the user may use the `dev_jll()` method inside of JLL packages
+to check out a mutable copy of the package to their `~/.julia/dev` directory.  An
+`override` directory will be created within the package directory, providing a
+convenient location for the user to copy in their own libraries over the typical
```

Comment:
```suggestion
convenient location for the user to copy in their own files over the typical
```

I'd also consider changing "binaries" at the start of this paragraph to "files", as in principle one might also want to tweak, say, a header or config file, right? Wrong?